### ISSUE TITLE
keras_parikh_entailment accuracy issue and always predicting same value 

### ISSUE THREAD

**Issue by tiru1930** [explosion/spaCy#1671]
I am able run this example with Insurance QA data set.I have made only one change where output labels instead of 3 i made it 2.
For InsuranceQA , i have sampled training data set to create labels, so there are total 27577 samples available.

When i train with this, training accuracy is 100% and dev accuracy is 100%, if evaluate on test data, all the time it is predicting the same value.

I am not able to find the issue, is this because of my data set??

## Info about spaCy

* **Platform:** Linux-4.10.0-38-generic-x86_64-with-Ubuntu-16.04-xenial
* **spaCy version:** 2.0.2
* **Models:** en, en_core_web_md
* **Python version:** 3.5.2

**Comment by tiru1930** [comment]
its data issue, closing
